.github/workflows/release.yaml

```diff
@@ -113,10 +118,66 @@ jobs:
           name: reth-rbuilder-${{ needs.extract-version.outputs.VERSION }}-${{ matrix.configs.target }}${{ matrix.features && '-' }}${{ matrix.features }}
           path: target/${{ matrix.configs.target }}/release/reth-rbuilder
 
+  cross-build-binary:
+    name: Cross-build binary
+    needs: extract-version
+    if: ${{ github.event.inputs.cross-build-binary == 'true' || github.event_name == 'push'}} # when manually triggered or version tagged
+    runs-on: warp-ubuntu-2404-x64-32x
+    container:
+      image: ubuntu:22.04
```

Comment:
I don't know what this is. The job runs in warp 24.04, right? What's this container for?

Reply:
mmm, so this will fail in a 22.04 container. I will try to run it in my fork as @metachris said.